### PR TITLE
clarification for thresh arg of dataset.dropna()

### PR DESCRIPTION
**Pull request by pierreloicq**

### CODE REVIEW

**Comment by mathause** [comment]
Thanks for the PR. Should that be `summed over`? The test failure should be unrelated.

**Comment by pierreloicq** [comment]
As you wish because I don't know, I am not native English speaker :-)

File: xarray/core/dataset.py [hunk 1]
```diff
@@ -5468,7 +5468,7 @@ def dropna(
             - all : if all values are NA, drop that label
 
         thresh : int or None, optional
-            If supplied, require this many non-NA values.
+            If supplied, require this many non-NA values (summed on all the subset variables).
```
Comment (mathause) [inline xarray/core/dataset.py:5471]:
```suggestion
            If supplied, require this many non-NA values (summed over all the subset variables).
```

**Comment by mathause** [comment]
This should merge on green - thanks for your contribution @pierreloicq!

**Pull request merged**